conda-forge.yml

```diff
@@ -6,6 +6,9 @@ build_platform:
   osx_arm64: osx_64
 conda_build:
   pkg_format: '2'
+conda_build_tool: "conda-build"
+conda_install_tool: "conda"
+conda_solver: "libmamba"
```

Comment:
Yep, ci-setup is going to nudge the solvers to obtain the "latest everything" but the conda-smithy config is still needed here because the default is still "mamba + mambabuild". In the (not so long!) future, when conda-build + conda-libmamba-solver is default we can omit this section.